app/root.tsx

```diff
@@ -66,7 +66,7 @@ export const links: LinksFunction = () => {
 export const meta: MetaFunction = ({ data, location }) => ({
   ...getSocialMetas({
     title: getMetaTitle(),
-    url: data.url,
+    url: data?.url ?? "",
```

Comment:
fyi that `data` will be undefined in meta functions when errors happen, throwing in a meta function will mess up error catching